### DISCUSSION TITLE
Add hover help for function arguments

### DISCUSSION THREAD

**Discussion by qpmnguyen** [posit-dev/positron#3773] [Feedback]
Hi all, 

I love working with Positron so far and it would be great if some of these features would be added in the future (if not already planned): 

First, adding the ability to pull in a summary of the functional argument documentation. For example in RStudio:
![image](https://github.com/posit-dev/positron/assets/25627274/80b4214d-5428-4556-af51-c150d360f432)

~Second, when using the `View()` function to see a data set, it would be great if variable labels (usually when importing data from SAS formats) could be visible below the columns. For example in RStudio: 
![image](https://github.com/posit-dev/positron/assets/25627274/5cc4161b-3415-401c-b228-5d3f21ad02f6)~
EDIT: Just saw that other people have requested this

**Comment by olivroy** [comment]
Is there plan to support argument-level help in the future in Positron? I really love being able to see help text for individual arguments!

**Comment by juliasilge** [comment]
I just opened https://github.com/posit-dev/positron/issues/5302 to track and prioritize this feature. Thanks!
